docker/main/rootfs/etc/s6-overlay/s6-rc.d/frigate/finish

```diff
@@ -25,4 +25,7 @@ elif [[ "${exit_code_service}" -ne 0 ]]; then
   fi
 fi
 
+# used by the docker healthcheck
```

Comment:
I made a big and silly mistake here. This cannot be created in finish script because it only gets invoked once Frigate python processes finishes finishing.

This file needs to be created as soon as Frigate python process starts its finishing logic, in Python.

Reply:
https://github.com/blakeblackshear/frigate/pull/18476